webapp/public/index.html

```diff
@@ -62,17 +52,18 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>
-      Give blood banks a lifeline. Register to donate life and get rewards from
```

Comment:
@JustinCast are you using prettier to ident this file? just to confirm

Reply:
Yes. Prettier with the included config in the repo.